01-react-hooks/README.md

```diff
@@ -0,0 +1,119 @@
+# React Hooks
+
+## What Are Hooks
+
+* Exciting new (experimental) feature in React that make usage of JS classes unnecessary
+* Hooks are functions that let you “hook into” React state and lifecycle methods from functional components
+* Don't be afraid of hooks - they are just more direct way to use the React features you already know
+* Until now, you had to choose between functional component vs. using lifecycle methods or state - you couldn't have both
+* Hooks make it possible to use state and lifecycle methods within functional component
+
+## Why To Use Hooks
+
+* Using only functional components with Hooks allows code-reuse that makes the application faster to develop and easier to maintain
```

Comment:
Are you showing this somehow in the exercises?
Would it be possible to address those points during coding? To point out "this is what I was talking about"?

Reply:
Good point - yes, I plan to stress this especially after the custom hook creation 👍